lib/md055.js

```diff
@@ -0,0 +1,40 @@
+// @ts-check
+
+"use strict";
+
+const { addError} = require("../helpers");
+
+const noLeadingPipeErrorMessage = "Table rows must start with a pipe";
+const noTrailingPipeErrorMessage = "Table rows must end with a pipe";
+
+module.exports = {
+    "names": ["MD055", "table-missing-border"],
+    "description": "Table is missing border pipes",
+    "information": new URL("https://github.github.com/gfm/#tables-extension-"),
```

Comment:
Please don't override this field. But it's a good callout that the GFM link should be added to the rule documentation, please!